monkey/monkey_island/cc/resources/auth/auth.py

```diff
@@ -74,6 +78,11 @@ def _create_access_token(username):
     return access_token
 
 
+def _check_attack_mitigations_in_mongo():
+    if AttackMitigations.COLLECTION_NAME not in mongo.db.list_collection_names():
+        init_collections()
```

Comment:
I feel like this function doesn't belong here since it has nothing to do with authentication's resource